src/connectivity/__mocks__/registration.ts

```diff
@@ -11,7 +11,7 @@ export const registrationParameters: __RegistrationParameters = {
 			language: "en",
 			platform: "windows",
 			platformVersion: "11",
-			version: "6.4"
+			version: "99.8.6.54321"
```

Comment:
Is this version change intentional?

Reply:
It looks strange, but it is, yes 😅 I wanted the default mock to be an absurdly high number so that the introduction of tests for new features didn't require future changes to the mock.